logic/settings.ts

```diff
@@ -584,6 +586,21 @@ export class NovelWordCountSettingTab extends PluginSettingTab {
 						})
 				);
 
+			new Setting(containerEl)
+				.setName("Exclude markdown footnotes")
```

Comment:
Let's change this to match the others:

```suggestion
				.setName("Exclude footnotes")
```

Reply:
Makes sense: it's markdown that Obsidian works with, goes without saying